source/_integrations/alexa.smart_home.markdown

```diff
@@ -6,23 +6,21 @@ ha_category:
   - Voice
 ha_release: "0.54"
 ---
+Amazon Alexa provides a Smart Home API for richer home automation control and supports the ability to use simple intuitive utterances without saying the skill name, such as:
 
-## Amazon Alexa Smart Home
+<kbd>
```

Comment:
A few reasons:

- When listing multiple input utterances as quotes. The markdown parser adds quotes to only to the beginning of the first line and the end of the last unless you separate each quote with a new line. This results in extra white space in the markdown, and in the final page. 
- In addition to the new lines, the resulting `<bockquote>` style increases the font-size 25% which took up significantly more space than I believed necessary for the information it's conveying to the reader. 
- While technically this could be considered a quote; semantically this is user input and should use `<kbd>` markup.

Ultimately I'll do whatever you want. I just wanted to provide the the resulting information in a cleaner semantically correct way to the user, and I didn't think quotes of code ticks did this.

Reply:
I appreciate that, however, we want to avoid and remove as many HTML (or any other magic) from our Markdown files as possible. Ideally, they should be just plain Markdown, making them portable.

Your suggestion defeats that.

I would rather suggest using a list, use italic and enquote the utterance. That will still give you a compact display and is till fully Markdown.

Reply:
Will do. Thanks for the suggestion.